gst/gst_message.go

```diff
@@ -84,6 +84,8 @@ func (m *Message) GetStructure() *Structure {
 		C.gst_message_parse_info_details((*C.GstMessage)(m.Instance()), (**C.GstStructure)(unsafe.Pointer(&st)))
 	case MessageWarning:
 		C.gst_message_parse_warning_details((*C.GstMessage)(m.Instance()), (**C.GstStructure)(unsafe.Pointer(&st)))
+	case MessageElement:
+		st = C.gst_message_get_structure((*C.GstMessage)(m.Instance()))
```

Comment:
on a second thought: the docs https://gstreamer.freedesktop.org/documentation/gstreamer/gstmessage.html?gi-language=c#gst_message_get_structure say that the returned structure needs a `transfer:none` I don't see that anywhere in this file. Am I missing something or does that need to be added?

Reply:
I believe that the ownership of the returned structure for `gst_message_get_structure` is consistent with `gst_message_parse_error_details`, `gst_message_parse_info_details` and `gst_message_parse_warning_details`. In all cases, the created Go gst.Structure will not own the C.GstStructure, and it will only remain valid until gst_structure_free is called (most likely by the GstMessage object). Since GstStructure is apparently not reference counted, the only way to ensure that the underlying structure has the same lifecycle as the Go object may be to copy it with `gst_structure_copy` (with the associated overhead). If we believe this copy is a needed improvement, can we address it in a different PR, as again, I believe the current PR is consistent with the existing code?